<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Monitor (Data-8bit, Stop-1bit, Parity-None)</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhMultiLine</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[STOP] Display</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>991</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>굴림체</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[CLOSE]
Port Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[CLEAR] Display</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_com_x">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_baudrate">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="SendTextEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Send">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OpenCloseComm">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[OPEN]</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_display_type">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HEX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DEC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ASCII</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Display CODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_FileSave">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE(capture.txt)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Send CODE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_send_type">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>31</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HEX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DEC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ASCII</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_endl">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0x03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0x0D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0x0A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CRLF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0x03 endl 0x02</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Line Feed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_UserLF">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User(HEX)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UserLF">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>31</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TCP_ServerName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_ServerClientSelect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CLIENT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SERVER</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="cb_TCP_Send">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP/IP CONN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TCP_PortNumber">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT#</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADDRESS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_SaveLogPeriodic">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Periodic Save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbRepeativeSend">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeative Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SendIntervalNumber">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
